README.md

```diff
@@ -39,6 +39,7 @@ An Electron-based developer tool for Lynx, providing mobile debugging features.
 - pnpm = 7.33.6
 - Git
 - Python3
+- python-is-python3
```

Comment:
I don't think this is required. Do you want to solve the issue of `python` on new versions of macOS? If so, I think the user needs to decide for themselves how to find the python environment, such as alias, instead of using this package.